### ISSUE TITLE
Low button contrast

### ISSUE THREAD

**Issue by HugoGiraudel** [morkro/happy-plants#54]
I suggest making the transparent black background from buttons a bit darker to improve contrast, especially when they overlap an image.

**Comment by morkro** [comment]
The buttons have been updated quite a while ago, addressing this issue.
